src/main/resources/application.properties

```diff
@@ -1,3 +1,13 @@
 spring.application.name=demo
 
+langchain4j.open-ai.chat-model.api-key=${OPENAI_API_KEY}
+langchain4j.open-ai.chat-model.model-name=gpt-4o-mini
+langchain4j.open-ai.chat-model.response-format=json_schema
```

Comment:
This setting ensures that [Structured Outputs](https://openai.com/index/introducing-structured-outputs-in-the-api/) feature is enabled and `BookExtractor` will automatically generate a JSON schema from `Book`/`Books` records and supply it to the LLM.